src/pudl/cli.py

```diff
@@ -60,6 +67,15 @@ def parse_command_line(argv):
     return arguments
 
 
+def get_etl_job():
```

Comment:
`reconstructable` used below requires a module level function that generates a job. I played around with `build_reconstructable_job` for more dynamic job creation, but found it's not too useful right now.